# clip
roi_x_min: 4.0
roi_x_max: 100.0
roi_y_min: -20.0
roi_y_max: 20.0
roi_z_min: -1.75
roi_z_max: 2.0

vehicle_x_min: -1.2 #转换到车辆坐标系下后，将车身点云切除，车辆坐标系中心为后轴中心０点
vehicle_x_max: 3.5
vehicle_y_min: -1.0
vehicle_y_max: 1.0
vehicle_z_min: -0.5
vehicle_z_max: 2.5

# Enable pointcloud downsampling via VoxelGrid filter
is_downsample: false
# downsampling leaf size
leaf_size: 0.1

# remove ground

cluster_method: 1  # 1:欧式聚类，2:DBSCAN聚类  3.Kmeans++  4.MeanShift  5.快速欧式聚类  6.暴力搜索欧式聚类

#  euclidean_cluster
cluster_tolerance: 0.5
min_cluster_size: 4
max_cluster_size: 10000
use_multiple_thres: true

#  dbscan_cluster
eps: 2
min_pts: 50

#  kmeans_cluster
kmeans_max_iteration: 50  # 最大迭代次数
kmeans_cluster_num: 20     # 聚类个数

#  meanshift_cluster
meanshift_bandwidth: 7.0    # 带宽参数
meanshift_max_iteration: 100 # 最大迭代次数
meanshift_convergence_threshold: 0.01 # 收敛阈值

#  fast_euclidean_cluster
fast_cluster_tolerance: 1.0  # 搜索半径
fast_cluster_max_size: 10     # 邻域搜索最大点数
fast_cluster_min_size: 10    # 聚类最小点数
fast_cluster_max_cluster: 50000  # 聚类最大点数

# boundingBox
in_estimate_pose: false
cluster_merge_threshold: 0.75